GitExtUtils/GitUI/TaskManager.cs

```diff
@@ -18,42 +18,36 @@ public TaskManager(JoinableTaskContext joinableTaskContext)
 
         public JoinableTaskFactory JoinableTaskFactory { get; init; }
 
-        public void FileAndForget(Task task, Func<Exception, bool>? fileOnlyIf = null)
+        public static async Task HandleExceptionsAsync(Func<Task> asyncAction, Func<Exception, Task> handleExceptionAsync)
```

Comment:
It'd be great if you could add appropriate xml-docs to each public method you're touching, this would help with the review as well as be helpful in the future.